packages/devtools_app/lib/src/shared/config_specific/import_export/import_export.dart

```diff
@@ -169,11 +157,12 @@ abstract class ExportController {
     // with other trace viewers (catapult, perfetto, chrome://tracing), which
     // require a top level field named "traceEvents".
     if (activeScreenId == ScreenMetaData.performance.id) {
-      final traceEvents = List<Map<String, dynamic>>.from(
-        contents[activeScreenId][traceEventsFieldName],
-      );
+      final activeScreen =
+          (contents[activeScreenId] as Map).cast<String, Object?>();
+      final traceEvents =
+          List.of((activeScreen[traceEventsFieldName] as List).cast<Object?>());
       contents[traceEventsFieldName] = traceEvents;
-      contents[activeScreenId].remove(traceEventsFieldName);
+      activeScreen.remove(traceEventsFieldName);
```

Comment:
gut-check: doest this write to `contents`? Or is `activeScreen` a new CastList instance whose updates won't apply to `contents[activeScreenId]`?

Reply:
This question still stands.